taxonomy/models.py

```diff
@@ -154,6 +154,61 @@ def __repr__(self):
         return '<CourseSkills id="{0}" skill="{1!r}">'.format(self.id, self.skill)
 
 
+class ProgramSkills(TimeStampedModel):
+    """
+    Skills that will be learnt by taking the program.
+
+    .. no_pii:
+    """
+
+    program_uuid = models.CharField(
```

Comment:
nit: Can you use [UUIDField](https://docs.djangoproject.com/en/4.0/ref/models/fields/#uuidfield) here?